src/extensions/ignite/remove-module.ts

```diff
@@ -1,14 +1,11 @@
-module.exports = (plugin, command, context) => {
+import { IgniteToolbox } from '../../types'
+
+export default (toolbox: IgniteToolbox) => {
   /**
    * Removes a npm-based module from the project.
-   *
-   * @param {string}  moduleName - The module name to remove.
-   * @param {Object}  options - Various uninstalling flags.
-   * @param {boolean} options.unlink - Should we unlink?
-   * @param {boolean} options.dev - is this a dev dependency?
    */
-  async function removeModule (moduleName, options = {}) {
-    const { print, system, ignite } = context
+  async function removeModule(moduleName: string, options: { unlink?: boolean; dev?: boolean } = {}) {
```

Comment:
You could do `options: { unlink = false, dev = false }` as well.

Reply:
![image](https://user-images.githubusercontent.com/1479215/52820139-0deadc00-3060-11e9-9350-de2a3da279ff.png)

Huh.